recipe/meta.yaml

```
@@ -1,43 +1,65 @@
-{% set version = "5.2.2" %}
+{% set name = "superlu" %}
+{% set version = "5.3.0" %}
 
 
 package:
-  name: superlu
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: superlu-{{ version }}.tar.gz
-  url: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_{{ version }}.tar.gz
-  sha256: 470334a72ba637578e34057f46948495e601a5988a602604f5576367e606a28c
+  url: https://github.com/xiaoyeli/{{ name }}/archive/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e464afa77335de200aeb739074a11e96d9bef6d0b519950cfa6684c4be1f350
+  patches:
+    # mc64 is under a comercial license.
+    # Patch taken from https://src.fedoraproject.org/rpms/SuperLU/raw/f38/f/superlu-removemc64.patch
+    - patches/superlu-removemc64.patch
 
 build:
   number: 0
   skip: true  # [win]
-  features:
+  ignore_run_exports:
+    - libgfortran5
+    - libgfortran
 
 requirements:
   build:
     - cmake
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - make
+    - patch
   host:
-    - libblas
-    - libcblas
+    - libopenblas {{ openblas }}
```

```suggestion
    - openblas-devel {{ openblas }}  # [blas_impl == 'openblas']
```
This includes the headers. And take care of the run_exports. This is aligned with other recipes such as numpy/scipy.